src/properties/peerlistwidget.cpp

```diff
@@ -170,9 +173,9 @@ void PeerListWidget::showPeerListMenu(const QPoint&)
   QAction *dlLimitAct = 0;
 #endif
   QAction *banAct = 0;
-  QAction *copyIPAct = 0;
+  QAction *copyPeersAct = 0;
   if (!selectedPeerIPs.isEmpty()) {
-    copyIPAct = menu.addAction(IconProvider::instance()->getIcon("edit-copy"), tr("Copy IP"));
+    copyPeersAct = menu.addAction(IconProvider::instance()->getIcon("edit-copy"), tr("Copy selected peers"));
```

Comment:
>Other items in the menu are in lowercase.

Ideally all our menus should have capitalized words. With the exception of words such as `a`, `an`, `to`, `the` etc
Now it depends on you for this **specific** code block(if you aren't bored):
1. either follow the current style aka don't capitalize your new menuitem
2. or capitalize all the menuitems

For the time being I think it is best to use "Copy selected" (or "Copy Selected")